src/app/dashboard/(admin)/admin/(settings)/settings/page.tsx

```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import axios from "axios";
-import { useSession } from "next-auth/react";
+import { signIn, useSession } from "next-auth/react";
```

Comment:
use the unstable_update

now imagine it is a google user how will you authenticate the user